### PR TITLE
libplatsupport/bcm2711: PL011 + mini UART support

### PR DESCRIPTION
**Pull request by lulu98**
Signed-off-by: Lukas Graber <lukas.graber@hensoldt-cyber.de>

### CODE REVIEW

File: libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h [hunk 1]
```diff
@@ -18,25 +18,35 @@
...
     NUM_CHARDEV,
     /* Aliases */
     PS_SERIAL0 = BCM2711_UART0,
+    PS_SERIAL1 = BCM2711_UART1,
+    PS_SERIAL2 = BCM2711_UART2,
+    PS_SERIAL3 = BCM2711_UART3,
+    PS_SERIAL4 = BCM2711_UART4,
+    PS_SERIAL5 = BCM2711_UART5,
     /* defaults */
-    PS_SERIAL_DEFAULT = BCM2711_UART0
```
Comment (kent-mcleod) [inline libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h:38]: Can you add to the commit message that this is aligning the device numbering with the device tree and that `BCM2711_UART0` was previously referring to the serial1 device.

**Review by kent-mcleod — APPROVED** [review]

File: libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h (hunk 1 above)
Comment (lulu98) [reply libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h:38]: Commit message contains the stated information now.

File: libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h [hunk 2]
```diff
@@ -18,25 +18,35 @@
  */
 #define UART_BUSADDR                0x7E215000
 
-#define UART_PADDR_0                (UART_BUSADDR-BUS_ADDR_OFFSET+PADDDR_OFFSET)
+#define UART_PADDR_1                (UART_BUSADDR-BUS_ADDR_OFFSET+PADDDR_OFFSET)
```
Comment (axel-h) [inline libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h:21]: Stragen that the style checker does not complain about the whitespaces here. Could you add them to imprve readability?
```suggestion
#define UART_PADDR_1                (UART_BUSADDR - BUS_ADDR_OFFSET + PADDDR_OFFSET)
```

**Review by axel-h — APPROVED** [review]

File: libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h (hunk 2 above)
Comment (lulu98) [reply libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h]: should be fixed now

**Pull request merged**